themes/ropensci/layouts/authors/terms.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+{{ partial "head.html" . }}
+</head>
+<body>
+{{ partial "header.html" . }}
+   <section>
+        <div class="container">
+          <div class="row">
+            <div class="col-offset-2 col-8 top-20">
+                <h2 style="text-align: center;">Authors</h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-8 col-offset-2 top-4">
+<ul>
+  {{ $data := .Data.Pages }}
+  {{ range $key, $value := .Data.Terms.ByCount }}
+      <a href="{{ $value.Name | urlize }}">{{ $value.Name }}</a>
```

Comment:
maybe something like https://gohugo.io/functions/replace/ could be used to remove the `-`  actually maybe this will do both things https://gohugo.io/functions/humanize/

Reply:
I saw the function but it would only capitalize the first name 😢

Reply:
Maybe https://github.com/gohugoio/hugo/issues/2455#issuecomment-247605248

Reply:
But having CSS in the html template does not sound like a good idea. 🤔